sources/platform/runtime/machine-image/ami-overview.md

```diff
@@ -28,7 +28,8 @@ You can click on any image below to see what is pre-installed:
 
 | Machine Image | Release date     |
 |---------------|-------------------|
-| [v6.6.4 (latest)](ami-v664/)     | June 28, 2018
+| [v6.7.4 (latest)](ami-v674/)     | Aug 2, 2018 |
+| [v6.6.4](ami-v664/)      | June 28, 2018 |
```

Comment:
why are we changing this?

Reply:
i dont get it
- the next release is going to be v6.7.2, not v6.7.4
- the page for v6.7.4 is not added in mkdocs.yml which means it wont be built 
- if we merge this, then the link will point to a missing page
am i missing anything here?